### DISCUSSION TITLE
LInk to HTML Version of the Email

### DISCUSSION THREAD

**Discussion by X-Raym** [hizzle-co/noptin#552] [Ideas]
Hi,

It would be nice to be able to provide a HTML web page version of the email, link on the bottom of the newsletter with a tag, so that if there is display issue in the user email client, he can check the newsletter content in a pretty way in a web browser.

I dont think it exist for now and it can be built with some custom URL query from the email to a PHP file on the site for custom render based on this query.

I also consider not having this as a pure version direct copy of the full html body, but maybe as shortcode on a regular wordpress page "newsletter last issue", but this is surely more specific. Just an the HTML version would be great by already.

Cheers !

**Comment by picocodes** [comment]
This can be implemented for normal newsletter emails. I've added a task to implement it.

**Comment by X-Raym** [comment]
@picocodes Cool ! how does it works ?
Could it works for automated newsletter?

**Comment by picocodes** [comment]
Hi @X-Raym,

- [x] https://noptin.com/guide/sending-emails/how-to-display-past-newsletter-issues-on-your-site/
- [x] https://noptin.com/guide/sending-emails/view-email-in-browser-link/

**Comment by X-Raym** [comment]
@picocodes Thanks !
